easybuild/easyconfigs/t/torchaudio/torchaudio-0.9.0-fosscuda-2020b-PyTorch-1.9.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonPackage'
+
+name = 'torchaudio'
+version = '0.9.0'
+local_pytorch_version = '1.9.0'
+versionsuffix = '-PyTorch-%s' % local_pytorch_version
+
+homepage = 'https://github.com/pytorch/audio'
+description = """ Data manipulation and transformation for audio signal
+processing, powered by PyTorch """
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+sources = [{
+    'filename': '%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/pytorch',
+        'repo_name': 'audio',
+        'tag': 'v%(version)s',
+        'recursive': True,
+    },
+}]
+
+checksums = [None]  # can't add proper checksum due to recursive cloning
+
+builddependencies = [
+    ('CMake', '3.18.4'),
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('PyTorch', local_pytorch_version),
+    ('SoX', '14.4.2'),
```

Comment:
Looks like `torchaudio` now includes its own and patched version of `SoX`, which gets built if `BUILD_SOX=1`: https://github.com/pytorch/audio/tree/v0.8.1/third_party

> BUILD_SOX: Deteremines whether build and bind libsox in non-Windows environments. (no effect in Windows as libsox integration is not available) Default value is 1 (build and bind). Use 0 for disabling it.

So, my impression is that we no longer need to include `SoX` in the dependencies. Could you confirm this, and if this is indeed the case, remove it from the dependencies?